js/flightlog_fields_presenter.js

```diff
@@ -700,20 +714,6 @@ function FlightLogFieldPresenter() {
                     'debug[2]':'Altitude',
                     'debug[3]':'Target altitude',
                 };
-                DEBUG_FRIENDLY_FIELD_NAMES.VTX_MSP = {
-                    'debug[all]': 'VTX MSP',
-                    'debug[0]': 'packetCounter',
-                    'debug[1]': 'isCrsfPortConfig',
-                    'debug[2]': 'isLowPowerDisarmed',
-                    'debug[3]': 'mspTelemetryDescriptor',
-                };
-                DEBUG_FRIENDLY_FIELD_NAMES.GPS_DOP = {
-                    'debug[all]': 'GPS Dilution of Precision',
-                    'debug[0]': 'Number of Satellites',
-                    'debug[1]': 'pDOP (positional - 3D)',
-                    'debug[2]': 'hDOP (horizontal - 2D)',
-                    'debug[3]': 'vDOP (vertical - 1D)',
-                };
```

Comment:
This is only available in firmware 4,4.

Reply:
True, but those lines are for when a debug has some of its names.  
Otherwise I think they all just go in the main list, regardless of when they are added.